Chap_API_Server.tex

```diff
@@ -185,18 +182,13 @@ \subsection{\code{PMIx_server_register_nspace}}
             \item \pastePRRTEAttributeItem{PMIX_SERVER_NSPACE}
             \item \pastePRRTEAttributeItem{PMIX_SERVER_RANK}
             \item \pastePRRTEAttributeItem{PMIX_NPROC_OFFSET}
-            \item \pastePRRTEAttributeItem{PMIX_APPLDR}
```

Comment:
APPLDR was already mentioned in the `required` section

Reply:
APPLDR seems to belong to the application-related section

Reply:
Yes, APPLDR represents the global rank of the first process of the given application. I think the confusion may perhaps be that we may not have used "job" and "application" separately enough - i.e., we may have interchanged them in some places. We can certainly read thru and ensure we are more careful about it.

Reply:
From my standpoint it is more than that, see my comment below about how to access the key of a specific application.